docs/api-overview.md

```
@@ -2869,6 +2871,120 @@ _Appears in:_
+GatewayCertSpec defines certificate management configuration
+
+
+
+_Appears in:_
+- [GatewaySpec](#gatewayspec)
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `type` _string_ | Type of certificate management: "user-provided" \| "cert-manager" \| "openshift-service-ca" | openshift-service-ca | Enum: [user-provided cert-manager openshift-service-ca] <br /> |
+| `secretRef` _[SecretReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#secretreference-v1-core)_ | Reference to user-provided certificate secret (when type="user-provided") |  |  |
+| `issuerRef` _[ObjectReference](#objectreference)_ | IssuerRef for cert-manager certificates (when type="cert-manager") |  |  |
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)